android/build.gradle

```diff
@@ -38,10 +39,12 @@ android {
                 signingConfig signingConfigs.debug
                 versionNameSuffix "/"+gitBranch+"/" + gitSha
                 applicationIdSuffix ".development"
+                manifestPlaceholders = [gcmPermissionRequired: "com.google.android.c2dm.permission.SEND"]
```

Comment:
This should be the empty string. The debugBlue should be a totally functional debug blue, except it uses prod styling.